src/response.pug

```diff
@@ -22,34 +22,39 @@ html(lang='it')
                                 h3.text-center Operazione in corso
                                 .text-muted.text-center.my-4 Ti preghiamo di attendere, senza chiudere la pagina.
                         .windowcont__response
-                            .windowcont__response__detail(data-response="3") <!-- TX_ACCEPTED -->
+                            .windowcont__response__detail(data-response="0") <!-- SUCCESS -->
```

Comment:
am I wrong or is some [status](https://pagopa.atlassian.net/wiki/spaces/I/pages/167216710/Mixpanel+Events#Verifica-payment-transaction---io-pay%2Fresponse.html) missing vs `PAYMENT_OUTCOME_CODE` ?

Reply:
good point, some `outcomes` are not mapped in the design and I think they cannot occur (they are errors between `PM` and `POS`). However it is a point to investigate and dedicate a separate PR.